### PR TITLE
Feature/endpoint hf timeline by se

### PR DESCRIPTION
**Pull request by cazapatamar**

### CODE REVIEW

File: src/routes/protected_areas.js [hunk 1]
```diff
@@ -352,6 +352,36 @@ module.exports = (errorHandler, paService) => {
       })
   )));
 
+  /**
+   * @apiGroup s_hf
+   * @api {get} /pa/:category/:se_type/hf/timeline SETimeLineInPA
+   * @apiName SETimeLineInPA
```
Comment (erikasv) [inline src/routes/protected_areas.js:358]: You didin't change the name `HFTimeLineInPA` as in the other areas

File: src/service/basin_subzone.js [hunk 2]
```diff
@@ -216,6 +216,25 @@ module.exports = (basinSubzonePersistence, seService) => {
...
 
+    /**
+     * Request a given strategic ecosystem HF timeline data inside a basin subzone
+     * @param {Number} subzoneId basin subzone id
+     * @param {String} seType strategic ecosystem type
+     *
+     * @return {Object} Object of HF values through time
+     */
+
+    getSEHFTimeline: async (eaId, seType) => {
```
Comment (erikasv) [inline src/service/basin_subzone.js:227]: the first parameter should be called `subzoneId`

File: src/service/state.js [hunk 3]
```diff
@@ -222,6 +222,25 @@ module.exports = (statePersistence, municipalityService, seService) => {
...
 
+    /**
+     * Request a given strategic ecosystem HF timeline data inside a state
+     * @param {Number} stateId state id
+     * @param {String} seType strategic ecosystem type
+     *
+     * @return {Object} Object of HF values through time
+     */
+
+    getSEHFTimeline: async (eaId, seType) => {
```
Comment (erikasv) [inline src/service/state.js:233]: The first parameter should be called `stateId`

File: src/service/basin_subzone.js (hunk 2 above)
Comment (cazapatamar) [reply src/service/basin_subzone.js]: Done

File: src/service/state.js (hunk 3 above)
Comment (cazapatamar) [reply src/service/state.js]: Done

File: src/routes/protected_areas.js (hunk 1 above)
Comment (cazapatamar) [reply src/routes/protected_areas.js:358]: Done

**Pull request merged**